src/export/exportFHIR.ts

```diff
@@ -0,0 +1,13 @@
+import { FHIRExporter } from './FHIRExporter';
+import { StructureDefinition } from '../fhirtypes';
+import { FSHTank } from '../import';
+
+/**
+ * Processes a set of FSH definitions into StructureDefinitions.
+ * @param {FSHTank} tank - the tank containing the FSJ definitions to process
+ * @returns {StructureDefinition[]} - the set of StructureDefinitions processed from the tank
+ */
+export function exportFHIR(tank: FSHTank): StructureDefinition[] {
```

Comment:
Again, in anticipation of needing to export other things -- and because we want to hold profiles and extensions separately, we probably want to return a structured class (e.g. `Package`).